ooc-ui.use

```diff
@@ -1,4 +1,12 @@
 Name: ooc UI
 Description: User interface stuff
 SourcePath: source/ui
+SourcePath: source/ui/x11
+SourcePath: source/ui/win32
```

Comment:
I don't think you need three `SourcePath`. `SourcePath: source/ui` would be enough.